config/crd/kustomization.yaml

```diff
@@ -16,14 +16,6 @@ resources:
 - bases/medusa.k8ssandra.io_medusabackupschedules.yaml
 #+kubebuilder:scaffold:crdkustomizeresource
 
-patches:
```

Comment:
This fixes #652: if you remove the `+kubebuilder:pruning:PreserveUnknownFields` annotation from `CassandraDatacenterTemplate`, this patch gets applied and removes everything in the config.